README.rst

```diff
@@ -406,6 +406,16 @@ Anything after ``python3 -`` will be passed as an argument to
 
 All arguments after -- will be passed to ``pip`` as options.
 
+OpenShift s2i (Source-To-Image)
+===============================
+
+micropipenv is shipped in UBI, Fedora and RHEL based container images. To
```

Comment:
This is not entirely true. If you enable it, it's installed from PyPI during the build of your container. Our plan for the future (next release of RHEL which will have micropipenv as an RPM) is to have it by default installed everywhere and use it as a default for Pipfiles.